src/app/core/services/fss-search-filter.service.ts

```diff
@@ -93,4 +94,29 @@ export class FssSearchFilterService {
     return filterExpression;
   }
 
+  getFilterExpressionForApplyFilter(fssFilterGroup: FilterGroup[]) {
+    var filterExpressionForApplyFilter = "";
+    fssFilterGroup.forEach(fg => {
+      var filterExpressionPerFilterGroup = "";
+      fg.items.forEach(item => {
+        if (item.selected === true) {
+          if (filterExpressionPerFilterGroup === "") {
+            filterExpressionPerFilterGroup = "$batchContains('" + item.title.replace(/'/g, "''") + "')";
```

Comment:
`item.title.replace(/'/g, "''")` is repeated later so lets break that out of this line and put it right after line 102 as a new variable `let formattedTitle = item.title.replace(/'/g, "''")`

then this becomes more readable and we don't repeat code.